app/src/main/java/com/duckduckgo/app/onboarding/ui/page/vpn/VpnPagesViewModel.kt

```diff
@@ -1,215 +0,0 @@
-import com.duckduckgo.app.onboarding.ui.page.vpn.VpnPagesViewModel.Action.LeaveVpnPermission
-import com.duckduckgo.app.onboarding.ui.page.vpn.VpnPagesViewModel.Action.OpenSettingVpnConflictDialog
-import com.duckduckgo.app.onboarding.ui.page.vpn.VpnPagesViewModel.Action.VpnPermissionDenied
-import com.duckduckgo.app.onboarding.ui.page.vpn.VpnPagesViewModel.Action.VpnPermissionGranted
-import com.duckduckgo.app.onboarding.ui.page.vpn.VpnPagesViewModel.Action.VpnPermissionResult
-import com.duckduckgo.app.onboarding.ui.page.vpn.VpnPagesViewModel.Command.RequestVpnPermission
-import com.duckduckgo.app.pixels.AppPixelName
-import com.duckduckgo.app.statistics.pixels.Pixel
-import com.duckduckgo.di.scopes.FragmentScope
-import com.duckduckgo.mobile.android.vpn.network.VpnDetector
-import com.duckduckgo.mobile.android.vpn.pixels.DeviceShieldPixels
-import com.duckduckgo.mobile.android.vpn.ui.onboarding.VpnStore
-import kotlinx.coroutines.channels.BufferOverflow.DROP_OLDEST
-import kotlinx.coroutines.channels.Channel
-import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.receiveAsFlow
-import kotlinx.coroutines.launch
-import javax.inject.Inject
-
-@ContributesViewModel(FragmentScope::class)
-class VpnPagesViewModel @Inject constructor(
-    private val pixel: Pixel,
-    private val vpnPixels: DeviceShieldPixels,
-    private val vpnDetector: VpnDetector,
-    private val vpnStore: VpnStore
-) : ViewModel() {
-
-    private val command = Channel<Command>(1, DROP_OLDEST)
-    internal fun commands(): Flow<Command> = command.receiveAsFlow()
-
-    private var viewHasShown: Boolean = false
-    private var lastVpnRequestTime = -1L
-
-    fun onAction(action: Action) {
-        when (action) {
-            IntroPageBecameVisible -> introPageBecameVisible()
-            PermissionPageBecameVisible -> permissionPageBecameVisible()
-            ContinueToVpnExplanation -> onContinueToVpnExplanation()
-            EnableVPN -> onCheckVpnPermission()
-            LeaveVpnIntro -> onLeaveVpnIntro()
-            LeaveVpnPermission -> onLeaveVpnPermission()
-            LearnMore -> onLearnMore()
-            DismissVpnConflictDialog -> vpnPixels.didChooseToDismissVpnConflictDialog()
-            OpenSettingVpnConflictDialog -> onOpenVpnSettings()
-            ContinueVpnConflictDialog -> onContinueVpnConflictDialog()
-            VpnPermissionGranted -> onVpnPermissionGranted()
-            is VpnPermissionDenied -> onVpnSystemPermissionDenied(action.intent)
-            is VpnPermissionResult -> onVPNSystemPermissionResult(action.resultCode)
-        }
-    }
-
-    private fun sendCommand(newCommand: Command) {
-        viewModelScope.launch {
-            command.send(newCommand)
-        }
-    }
-
-    private fun introPageBecameVisible() {
-        if (!viewHasShown) {
-            viewHasShown = true
-            pixel.fire(AppPixelName.ONBOARDING_VPN_INTRO_SHOWN)
```

Comment:
Should we clean up `ONBOARDING_VPN_INTRO_SHOWN` from AppPixelName?

Other pixels in the same category:
`ONBOARDING_VPN_PERMISSION_SHOWN`
`ONBOARDING_VPN_INTRO_CONTINUED`
`ONBOARDING_VPN_INTRO_SKIPPED`
`ONBOARDING_VPN_PERMISSION_SKIPPED`
`ONBOARDING_VPN_PERMISSION_FAQ_LAUNCHED`
`ONBOARDING_VPN_PERMISSION_CONTINUED`
`ONBOARDING_VPN_PERMISSION_LAUNCHED`
`ONBOARDING_VPN_PERMISSION_DENIED`

Reply:
Will do!